WikibaseEnitiyStatementReomover.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace Addwiki\Commands\Wikibase;
+
+use ArrayAccess;
+use Asparagus\QueryBuilder;
+use DataValues\Deserializers\DataValueDeserializer;
+use DataValues\Serializers\DataValueSerializer;
+use GuzzleHttp\Client;
+use Mediawiki\Api\ApiUser;
+use Mediawiki\Api\MediawikiApi;
+use RuntimeException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Wikibase\Api\WikibaseFactory;
+use Wikibase\DataModel\Entity\PropertyId;
+
+/**
+ * @author Addshore
+ * @author Lucie-Aimée Kaffee
+ */
+class WikidataEntityStatementRemover extends Command {
+
+	private $appConfig;
+
+	/**
+	 * @var WikibaseFactory
+	 */
+	private $wikibaseFactory;
+
+	/**
+	 * @var MediawikiApi
+	 */
+	private $wikibaseApi;
+
+	/**
+	 * @var SparqlQueryRunner
+	 */
+	private $sparqlQueryRunner;
+
+	public function __construct( ArrayAccess $appConfig ) {
+		$this->appConfig = $appConfig;
+
+		$defaultGuzzleConf = array(
+			'headers' => array( 'User-Agent' => 'addwiki - Wikibase Reference As Statement Remover' )
+		);
+		$guzzleClient = new Client( $defaultGuzzleConf );
+		$this->sparqlQueryRunner = new SparqlQueryRunner( $guzzleClient );
+
+		$this->wikibaseApi = new MediawikiApi( "https://www.wikidata.org/w/api.php" );
```

Comment:
As this is wikibase generic there should be a wiki parameter and the URL of that site should be used rather than hardcoding wikidata!